scripting/include/cbasenpc/nextbot/behavior.inc

```diff
@@ -106,71 +118,142 @@ typeset ActionEventResponderCallback
 	// OnTerritoryCaptured (territoryID = victim)
 	// OnTerritoryLost (territoryID = victim)
 	function int (NextBotAction action, int actor, int victim);
+	function int (NextBotAction action, CBaseCombatCharacter actor, CBaseEntity victim);
 	function void (NextBotAction action, int actor, int victim);
+	function void (NextBotAction action, CBaseCombatCharacter actor, CBaseEntity victim);
 
 	// OnAnimationEvent
 	function int (NextBotAction action, int actor, int event);
+	function int (NextBotAction action, CBaseCombatCharacter actor, int event);
+
 	function void (NextBotAction action, int actor, int event);
+	function void (NextBotAction action, CBaseCombatCharacter actor, int event);
+
 	function int (NextBotAction action, int actor, int event, const char[] options);
+	function int (NextBotAction action, CBaseCombatCharacter actor, int event, const char[] options);
+
 	function void (NextBotAction action, int actor, int event, const char[] options);
+	function void (NextBotAction action, CBaseCombatCharacter actor, int event, const char[] options);
+
 	function int (NextBotAction action, int actor, int event, const char[] options, float cycle, float eventtime);
+	function int (NextBotAction action, CBaseCombatCharacter actor, int event, const char[] options, float cycle, float eventtime);
+
 	function void (NextBotAction action, int actor, int event, const char[] options, float cycle, float eventtime);
+	function void (NextBotAction action, CBaseCombatCharacter actor, int event, const char[] options, float cycle, float eventtime);
+
 	function int (NextBotAction action, int actor, int event, const char[] options, float cycle, float eventtime, int eventtype);
+	function int (NextBotAction action, CBaseCombatCharacter actor, int event, const char[] options, float cycle, float eventtime, int eventtype);
+
 	function void (NextBotAction action, int actor, int event, const char[] options, float cycle, float eventtime, int eventtype);
-	function int (NextBotAction action, int actor, int event, const char[] options, float cycle, float eventtime, int eventtype, int eventSourceEntity);

```

Comment:
Is this intended, otherwise that's a breaking change.

Reply:
Nope, that's a mistake. Good catch!